src/pages/documentation/release-notes/index.md

```diff
@@ -6,6 +6,14 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 ## Aug 8, 2023
 
+### iOS Places 4.1.0 and Android Places 2.1.0
+
+* Support for forwarding location entry and exit events to Adobe Experience Platform.
+
+For more information, refer to the documentation below:
+
+* [Places Service event forwarding to Adobe Experience Platform](/places/places-to-platform.md)
```

Comment:
Should we consolidate this in one line, reading:

- Added support for forwarding location entry and exit events to Adobe Experience Platform. For more information, refer to the [Places Service event forwarding to Adobe Experience Platform](/places/places-to-platform.md) guide.

Reply:
Crisp. Thanks!